docs/core/tools/dotnet-help.md

```diff
@@ -38,10 +38,6 @@ Prints out a short help for the command.
 
 ## Examples
 
-List the project references for the specified project:
+Opens the documentation page for the [dotnet new](dotnet-new.md) command:
 
-`dotnet list app/app.csproj reference`
-
-List the project references for the project in the current directory:
-
-`dotnet list reference`
+`dotnet help new`
```

Comment:
Actually, all of our CLI examples are inline code. So to be consistent, I'd have to change all others. We currently don't a language identifier for .NET Core CLI. I can make a test on a separate branch to see which one looks better.